core/cmd/remote_client.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/smartcontractkit/chainlink/core/web"
 )
 
-var errUnauthorized = errors.New("401 Unauthorized")
+var errUnauthorized = errors.New(http.StatusText(http.StatusUnauthorized))
```

Comment:
Seems like it would be good to include an explanation about how to log in to the node (something about `chainlink admin login`)